<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Relacionar</class>
 <widget class="QMainWindow" name="Relacionar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relacionar Tablas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Niagara Solid&quot;;</string>
    </property>
    <property name="text">
     <string>Imprtar Excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_abrir">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Excel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Niagara Solid&quot;;</string>
    </property>
    <property name="text">
     <string>Excel Actual</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 38pt &quot;MS Serif&quot;;
</string>
    </property>
    <property name="text">
     <string>Relacionar Tabla</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Columna a relacionar</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Columna a relacionar</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_colA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_colN">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Nombre de la nueva columna </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="T_col">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Valor a buscar columna</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Box_buscar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_previsalizar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Previsualizar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Vista">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>771</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>Continuar ---&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione Hoja</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
